composer.json

```diff
@@ -15,8 +15,8 @@
   ],
   "require": {
     "php": ">=7.0",
-    "illuminate/support": "~5.5.0",
-    "illuminate/view": "~5.5.0"
+    "illuminate/support": "~5.3.0",
+    "illuminate/view": "~5.3.0"
```

Comment:
~5.3.0 means >=5.3.0 and <5.4.0
I think, we need to use ^5.3 constraint

Reply:
Yes!